build/makelib/golang.mk

```diff
@@ -170,7 +170,13 @@ go.mod.update:
 
 .PHONY: go.mod.check
 go.mod.check:
-	@echo === ensuring modules are tidied
+	@echo === syncing root modules with APIs modules
+	@cp -a go.sum pkg/apis/go.sum
+	@cat go.mod | sed -e 's|^module github.com/rook/rook|module github.com/rook/rook/pkg/apis|' \
```

Comment:
`sed` is always fun on Mac, BSD vs GNU sed, I just remembered this script in the Rook repo which we use to workaround this situation: https://github.com/rook/rook/blob/7627a92319cd8797c4feceaad9d43ba9b3ad5a08/build/sed-in-place

Reply:
Mac is so unexpected :)